src/main/resources/templates/staySafe.html

```diff
@@ -4,17 +4,13 @@
 <!-- Required meta tags -->
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="styles.css"> 
 
-<!-- Required meta tags -->
```

Comment:
I deleted these repeated meta tags to reduce 5 lines of unnecessary code in the html.